xcode/Subconscious/Shared/Components/Detail/UserProfileDetailView.swift

```diff
@@ -975,7 +918,7 @@ struct UserProfileDetailModel: ModelProtocol {
         
         return update(
             state: state,
-            action: .refresh(forceSync: true),
+            action: .refresh(forceSync: false),
```

Comment:
Intentional?

Reply:
Yep, this is handled by the AppStore reacting to these.